aicsimageio/readers/czi_reader.py

```diff
@@ -183,6 +188,27 @@ def scenes(self) -> Tuple[str, ...]:
                 # If the scene is implicit just assign it name Scene:0
                 if len(scene_names) < 1:
                     scene_names = [metadata_utils.generate_ome_image_id(0)]
+                else:
+                    # reconcile scene list against the dims shape
+                    dims_shape = czi.get_dims_shape()
+                    if len(scene_names) != len(dims_shape) and czi.shape_is_consistent:
+                        dims_shape_dict = dims_shape[0]
+                        scene_range = dims_shape_dict.get(CZI_SCENE_DIM_CHAR)
+                        if scene_range is not None:
+                            scene_names = scene_names[scene_range[0] : scene_range[1]]
```

Comment:
Will this basically "fix" the scene list?

For the file that caused the bug, will these two lines make it so that the only scenes reported back to `img.reader.scenes` is the one that is valid?

Reply:
Yes this is where if you have a file that is a child file of a split multiscene czi,  it has metadata info about all N scenes but the file really only contains a subset of them.  It should not ever let you set_scene to one that is outside that range, so the reader will only seem to recognize the scenes for which it has pixel data.

Reply:
Perfect!